src/App.jsx

```diff
@@ -1,3 +1,32 @@
+import {
+  Switch,
+  Route,
+} from 'react-router-dom';
+import Login from './views/Login';
+import EntryList from './views/EntryList';
+import PrivateRoute from './components/PrivateRoute';
+import UserProvider from './context/UserContext';
+import Header from './components/Header';
+
+
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <UserProvider>
```

Comment:
I've found it easier when testing `App` to have my context provider in `index.jsx`  with my `router`